caiman/motion_correction.py

```diff
@@ -2706,10 +2706,15 @@ def motion_correct_batch_rigid(fname, max_shifts, dview=None, splits=56, num_spl
             save_movie = save_movie_rigid
             logging.debug('saving!')
 
+
+        if isinstance(fname, tuple):
+            base_name=os.path.split(fname[0])[-1][:-4] + '_rig_'
```

Comment:
Could use a comment to make it clear what this (not very legible) transformation is doing